src/tests/testAccomod.spec.js

```diff
@@ -0,0 +1,177 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import fs from 'fs';
+import path from 'path';
+import { describe, it } from 'mocha';
+import myserver from '../index';
+import mock2 from './mockData/loginMockData';
+import mock from './mockData/accommodationData';
+import helper from '../utils/helper';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const wrongFile = fs.readFileSync(path.join(__dirname, 'mockData/profile.js'));
+
+let loggedtoken = '';
+describe('Update/delete an accommodation tests', () => {
+  before((done) => {
+    chai.request(myserver)
+      .post('/api/v1/users/login')
+      .send(mock2[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(201);
+        loggedtoken = res.body.data.token;
+        done();
+      });
+  });
+  it('It should return a 200 status, when an accommodation has been updated', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .set('Authorization', loggedtoken)
+      .send(mock[0])
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        done();
+      });
+  });
+  it('It should return a 400 status, when wrong file image is selected', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .set('Authorization', loggedtoken)
+      .field('address', 'kigali')
+      .field('name', 'hotel')
+      .field('description', 'the first hotel in region')
+      .field('cost', '200000')
+      .field('availableSpace', 'rooms and pool')
+      .field('services', 'gym, breakfast')
+      .field('amenities', 'wifi, tv')
+      .attach('images', wrongFile, 'profile.js')
+      .end((err, res) => {
+        expect(res.status).to.equal(400);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
+        expect(res.body.msg).to.equals('uploaded files should be images');
+        done();
+      });
+  });
+  it('It should return a 200 status, when all data are note updated', (done) => {
+    chai.request(myserver)
+      .patch('/api/v1/accommodations/1')
+      .set('Authorization', loggedtoken)
+      .field('address', 'kigali')
+      .field('name', 'hotel')
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('msg');
+        expect(res.body).to.have.property('status');
```

Comment:
Assert that the fields are actually updated. `expect that address if 'kigali'`, etc. Also for name, and other test cases